app/views/inflation_proof.scala.html

```diff
@@ -29,84 +32,72 @@
 <script type="text/javascript" src='@controllers.routes.AssetsController.at("javascripts/progressive-reveal.js")'></script>
 <script type="text/javascript" src='@controllers.routes.AssetsController.at("javascripts/revaluation-date.js")'></script>
 }
+@dateHtml = {
+    @govukDateInput(DateInput(
+        id = "revaluationDate",
+        hint = Some(Hint(
+          content = Text(Messages("gmp.date.example"))
+        )),
+        fieldset = Some(Fieldset(
+            legend = Some(Legend(
+                content = Text(Messages("gmp.inflation_proofed")),
+                classes = "govuk-fieldset__legend--m",
+                isPageHeading = true
+            ))
+        )),
+        items = Seq(
+            InputItem(
+                name = "revaluationDate.day",
+                classes = "govuk-input--width-2",
+                label=Some("Day")),
+            InputItem(
+                name = "revaluationDate.month",
+                classes = "govuk-input--width-2",
+                label=Some("Month")),
+            InputItem(
+                name = "revaluationDate.year",
+                classes = "govuk-input--width-4",
+                label=Some("Year"))),
+    ))
+
+<hr class="govuk-section-break govuk-section-break--m" aria-hidden = true />
+<p class="govuk-body">@Messages("gmp.inflationproof.subtext")</p>
+}
+
+
 
 @gmpMain(title = Messages("gmp.inflation_proof.question") + " - " + Messages("service.title") + " - " + Messages("gov.uk"), pageScripts = Some(pageScripts)) {
 
 
     @includes.back_link(Messages("gmp.inflation_proof.question"))
 
-    @views.html.helpers.errorSummary(form = inflationProofForm, focusId = {if(!inflationProofForm.hasGlobalErrors && inflationProofForm.error("revaluate").isDefined) Some("revaluate-revaluation-date") else Some("revaluationDate_day")})
 
         @viewHelpers.form(action = routes.InflationProofController.post) {
+            @if(inflationProofForm.errors.nonEmpty){
+                @govukErrorSummary(ErrorSummary().withFormErrorsAsText(inflationProofForm))
```

Comment:
We have to print individual errors as per new dat component for example if day is missing, the error should say day is missing and highlight the day field in red. if month and year are missing , then it should say the same and highlight those fields in red